src/pnc-download/convertXls.test.ts

```diff
@@ -5,7 +5,7 @@ const expectedCCJSOffenceCodesFile = "test-data/expected-output/pnc-ccjs-expecte
 const expectedACPOOffenceCodesFile = "test-data/expected-output/pnc-acpo-expected.json"
 
 describe("convertXls", () => {
-  it("should convert PNC CCJS file", () => {
+  it.skip("should convert PNC CCJS file", () => {
```

Comment:
would this work with the expected result changed to use native types? If not I think leaving skipped tests in the file clutters it up, they'll be in git if we need them in future

Reply:
No, it is the same behaviour we just never run these tests in the CI, I'll try fixing them instead!

Reply:
Fixed the test in this PR https://github.com/ministryofjustice/bichard7-next-data/pull/37